src/index.html

```diff
@@ -8,6 +8,79 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="stars">
+      <div class="stars--0">
+        <div class="box">
+        </div>
+        <div class="box">
+        </div>
+        <div class="box">
+        </div>
+        <div class="box">
+        </div>
+        <div class="box">
+        </div>
+      </div>
+      <div class="stars--1">
+        <div class="box">
```

Comment:
This element should be `stars__star` not `box`.
![image](https://user-images.githubusercontent.com/25587299/221599994-8c87ea75-4c1a-455f-ac4f-89c63a4a0b7c.png)